recipes/rb-cool.io/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cool.io" %}
+{% set version = "1.5.4" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 35fce51e731e56052ae63a180aa6f01de3677ff80a2382934085e687c7370351
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+    - make -C {{ PREFIX }}/lib/ruby/gems/{{ ruby }}.0/gems/{{ name }}-{{ version }}/ext/{{ name }} clean
+  skip: {{ win }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - ruby
+  run:
+    - ruby
```

Comment:
It looks like both `cool.io` and `nio4r` have a dependency on `libev` and are venturing the libraries. 

Lets wait until #9450 is fixed to understand how to properly unvendor libev from nio4r and apply the same technique to your package.

Reply:
@conda-forge/staged-recipes, @conda-forge/core

What is the guidance when vendoring patched versions of C libraries? 
In the case of `rb-cool.io`, it vendors a [patched](https://github.com/tarcieri/cool.io/blob/0fd3fd1d8e8d81e24f79f809979367abc3f52b92/libev_ruby_gil.diff#L113-L145) version of `libev`  to solve an incompatibility between Ruby's global-interpreter-lock and libev's concurrency management.

Reply:
If it's absolutely necessary, then the vendored library and the original library must be co-installable and vendored library's license file must be added as a license in this package.

Reply:
Fair, any guidelines on how I can ensure both libraries can be co-installed?

Reply:
Does this package install a `libev.so`?

Reply:
It does not look like it does, instead they include `libev.c` in their own extension. see 

https://github.com/tarcieri/cool.io/blob/master/ext/cool.io/libev.c

I am building the package locally to verify. I imagine, that if no libev.so is created that they can both be installed without conflicts, is that correct?

Reply:
yes